.environment/eslint/run-eslintCheck.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+function note() {
+    echo "eslint-check> info: ${*}"
+}
+
+function js_fmt_check() {
+    note "Checking JavaScript formatting."
+    MODIFIED_TSX_FILES_COUNT=$(git status --porcelain | grep "\.tsx$" | wc -l)
+    MODIFIED_JS_FILES_COUNT=$(git status --porcelain | grep "\.js$" | wc -l)
+    if [[ ${MODIFIED_TSX_FILES_COUNT?} != 0 ]] || [[ ${MODIFIED_JS_FILES_COUNT?} != 0 ]]; then
+        cd frontend-react
+        npm run lint
```

Comment:
It looks like you need to have run `yarn install` to run `npm run lint` -- but of course if you're modifying JS code then you should already have run that. So I guess it's ok :-)

Reply:
couldn't hurt!